### PR TITLE
hw-mgmt: thermal: Disable kernel thermal algorithm on liquid cooled s…

### PR DESCRIPTION
**Pull request by felixradensky**
…ystems

Disable kernel thermal algorithm on MQM9510 and MQM9520 liquid cooled systems.

### CODE REVIEW

**Comment by felixradensky** [comment]
RERUN

**Review by ciju-nvidia — DISMISSED** [review]

File: usr/usr/bin/hw-management-liquid-cooling.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-liquid-cooling.sh:1]: Nee to add license.

Comment (felixradensky) [reply usr/usr/bin/hw-management-liquid-cooling.sh:1]: Added license.

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by felixradensky** [comment]
RERUN

**Comment by felixradensky** [comment]
RERUN

**Pull request merged**